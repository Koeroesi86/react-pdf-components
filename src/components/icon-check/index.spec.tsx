import React from "react";
import { renderToString } from "@react-pdf/renderer";
import { Wrapper } from "@test-helpers";
import IconCheck from "./index";

describe("<IconCheck/>", () => {
  it("should render", async () => {
    const result = await renderToString(
      <Wrapper>
        <IconCheck width={12} height={12} color="#000"/>
      </Wrapper>
    );
    expect(result).toMatchSnapshot();
  });
});
