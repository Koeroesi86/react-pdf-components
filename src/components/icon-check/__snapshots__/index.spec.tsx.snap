// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IconCheck/> should render 1`] = `
"%PDF-1.3
%����
7 0 obj
<<
/Type /Page
/Parent 1 0 R
/MediaBox [0 0 595.280029 841.890015]
/Contents 5 0 R
/Resources 6 0 R
>>
endobj
6 0 obj
<<
/ProcSet [/PDF /Text /ImageB /ImageC /ImageI]
>>
endobj
5 0 obj
<<
/Length 608
>>
stream
1 0 0 -1 0 841.890015 cm
q
q
1 0 0 1 0 0 cm
q
q
0 3 m
12 3 l
12 3 12 3 12 3 c
12 15 l
12 15 12 15 12 15 c
0 15 l
0 15 0 15 0 15 c
0 3 l
0 3 0 3 0 3 c
h
W n
1 0 0 1 0 3 cm
0.023438 0 0 0.023438 0 0 cm
1 0 0 1 0 0 cm
q
/DeviceRGB cs
0 0 0 scn
173.898 439.404 m
7.498 273.004 l
-2.499 263.007 -2.499 246.798 7.498 236.8 c
43.701 200.596 l
53.698 190.598 69.908 190.598 79.905 200.596 c
192 312.69 l
432.095 72.596 l
442.092 62.599 458.302 62.599 468.299 72.596 c
504.502 108.8 l
514.499 118.797 514.499 135.006 504.502 145.004 c
210.102 439.405 l
200.104 449.402 183.895 449.402 173.898 439.404 c
h
f
Q
Q
Q
Q
Q

endstream
endobj
9 0 obj
(react-pdf)
endobj
10 0 obj
(react-pdf)
endobj
11 0 obj
(D:20210414183756Z)
endobj
8 0 obj
<<
/Producer 9 0 R
/Creator 10 0 R
/CreationDate 11 0 R
>>
endobj
3 0 obj
<<
/Type /Catalog
/Pages 1 0 R
/Names 2 0 R
/ViewerPreferences 4 0 R
>>
endobj
1 0 obj
<<
/Type /Pages
/Count 1
/Kids [7 0 R]
>>
endobj
2 0 obj
<<
/Dests <<
  /Names [
]
>>
>>
endobj
4 0 obj
<<
/DisplayDocTitle true
>>
endobj
xref
0 12
0000000000 65535 f 
0000001111 00000 n 
0000001168 00000 n 
0000001024 00000 n 
0000001215 00000 n 
0000000200 00000 n 
0000000133 00000 n 
0000000015 00000 n 
0000000950 00000 n 
0000000859 00000 n 
0000000886 00000 n 
0000000914 00000 n 
trailer
<<
/Size 12
/Root 3 0 R
/Info 8 0 R
>>
startxref
1258
%%EOF
"
`;
